### DISCUSSION TITLE
建议在Wiki中增加macOS信任crx的方法(附方法)

### DISCUSSION THREAD

**Discussion by RalphTsui** [ronggang/PT-Plugin-Plus#712] [Ideas]
参考：https://xclient.info/a/1ddd2a3a-d34b-b568-c0d0-c31a95f0b309.html
通过安装描述文件的方式来信任
本人已经测试通过~ 使用稳定~

**Comment by Rhilip** [comment]
我觉得是不是先转成 discussion 好些，或者能否提供 markdown文件，我来设置wiki？
